helix-term/src/application.rs

```diff
@@ -19,9 +19,12 @@ use crossterm::{
     execute, terminal,
 };
 #[cfg(not(windows))]
-use signal_hook::{consts::signal, low_level};
-#[cfg(not(windows))]
-use signal_hook_tokio::Signals;
+use {
+    signal_hook::{consts::signal, low_level},
+    signal_hook_tokio::Signals,
+};
+#[cfg(windows)]
+type Signals = futures_util::stream::Empty<()>;
```

Comment:
Ah, I didn't know you mean like this. I thought you just do `futures_util::stream::Empty` below instead.